### ISSUE TITLE
Binning

### ISSUE THREAD

**Issue by quasiben** [blaze/dask#334]
Part of issue #333 would probably require some kind of binning?  I do and see a lot of the following:

```
In [173]: df1.sepal_length.head()
Out[173]:
0    5.1
1    4.9
2    4.7
3    4.6
4    5.0
Name: sepal_length, dtype: float64

In [174]: df1.sepal_length.value_counts(bins=3)
Out[174]:
5.5000    71
4.2964    59
6.7000    20
dtype: int64
```

Can dask similarly do binning with optional bin sizes over most numerical dtypes?

**Comment by cpcloud** [comment]
whoa i didn't even know `bins` was an argument

**Comment by quasiben** [comment]
This should be the equivalent, correct?

```
np.histogram(df1.sepal_length.values, bins=10)
```

**Comment by mrocklin** [comment]
This is possible but requires two passes over the data or a very clever algorithm.  My guess is that we don't want to think about exact binning but more about approximate quantiles.

**Comment by jcrist** [comment]
Closed by #675.
